app/users/edit/credits/index/template.hbs

```diff
@@ -22,53 +33,71 @@
           <LoadingIcon />
         </div>
       {{else}}
-        {{#with this.absenceCredits.lastSuccessful.value as |absenceCredit|}}
-          <div class="card-block">
-            <div class="grid">
-              <div class="grid-cell">
-                <table class="table table--striped">
-                  <thead>
-                    <tr>
-                      <th>Valid as of</th>
-                      <th>Days</th>
-                      <th>Type</th>
-                      <th>Comment</th>
-                    </tr>
-                  </thead>
-                  <tbody>
-                    {{#each absenceCredit as |absenceCredit|}}
-                      <tr class="{{if (can 'edit absence-credit') 'pointer'}}" {{action (perform this.editAbsenceCredit) absenceCredit.id on='click'}}>
-                        <td>{{moment-format absenceCredit.date 'DD.MM.YYYY'}}</td>
-                        <td>{{absenceCredit.days}}</td>
-                        <td>{{absenceCredit.absenceType.name}}</td>
-                        <td>{{absenceCredit.comment}}</td>
+        {{#let this.absenceCredits.lastSuccessful.value as |absenceCredits|}}
+          {{#if absenceCredits}}
+            <div class="card-block">
+              <div class="grid">
+                <div class="grid-cell">
+                  <table class="table table--striped">
+                    <thead>
+                      <tr>
+                        <th>Valid as of</th>
+                        <th>Days</th>
+                        <th>Type</th>
+                        <th>Comment</th>
                       </tr>
-                    {{/each}}
-                  </tbody>
-                </table>
+                    </thead>
+                    <tbody>
+                      {{#each absenceCredits as |absenceCredit|}}
+                        <tr
+                          class="{{if (can 'edit absence-credit') 'pointer'}}"
```

Comment:
No need for  quotes if the value is a helper which returns a string.
```suggestion
                          class={{if (can 'edit absence-credit') 'pointer'}}
```

Reply:
`done`